examples/performance/01_gpu.ipynb

```diff
@@ -307,8 +307,7 @@
       "    {\n",
```

Comment:
<p>so there is diff here, no temp?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>